packages/node_modules/@ciscospark/react-redux-spark/src/__snapshots__/reducer.test.js.snap

```diff
@@ -2,58 +2,58 @@
 
 exports[`spark reducer should handle REGISTER_DEVICE_FAILURE 1`] = `
 Immutable.Map {
-  "error": [Error: device failure],
-  "status": Immutable.Map {
-    "authenticated": false,
-    "authenticating": false,
-    "registered": false,
-    "registerError": true,
-    "registering": false,
+  error: [Error: device failure],
+  status: Immutable.Map {
+    authenticated: false,
+    authenticating: false,
+    registered: false,
+    registerError: true,
+    registering: false,
   },
-  "spark": null,
+  spark: null,
 }
 `;
 
 exports[`spark reducer should handle STORE_SPARK_INSTANCE 1`] = `
 Immutable.Map {
-  "error": null,
-  "status": Immutable.Map {
-    "authenticated": false,
-    "authenticating": false,
-    "registered": false,
-    "registerError": false,
-    "registering": false,
+  error: null,
+  status: Immutable.Map {
+    authenticated: false,
+    authenticating: false,
+    registered: false,
+    registerError: false,
+    registering: false,
   },
-  "spark": Object {
+  spark: Object {
     "mock": true,
   },
 }
 `;
 
 exports[`spark reducer should handle UPDATE_SPARK_STATUS 1`] = `
 Immutable.Map {
-  "error": null,
-  "status": Immutable.Map {
-    "authenticated": true,
-    "authenticating": false,
-    "registered": true,
-    "registerError": false,
-    "registering": false,
+  error: null,
```

Comment:
This doesn't look quite right.... There was a change in jest or Immutable that moved from no quotations around key names to with quotations. Can you check to make sure you're up to date with jest and immutable and run `jest` again?

Reply:
Forgot to `npm i` after merging. Fixing now.